about-us.html

```diff
@@ -186,9 +186,7 @@ <h1>About Us</h1>
               another priority. Nicolas Dorier, founder of the BTCPay Server Project, shall possess two votes for the
               purpose of breaking a tie.</p>
 
-            <p>BTCPay Server Foundation is not a
-              501(c)(3) organisation. If you have questions about the tax consequences of donating, please consult a tax
-              professional.</p>
+            <p>BTCPay Server Foundation is a non-profit organization.
```

Comment:
Trying not to hijack :) Just needs a closing `</p>` after `organisation.`

Reply:
My bad, should be good now @vswee